website/docs/handle_exceptions.md

```diff
@@ -4,46 +4,88 @@ title: handle_exceptions Decorator
 sidebar_label: handle_exceptions Decorator
 ---
 
-## handle_exceptions Decorator
-`Core-Lib`'s `web_helpers` provides `handle_exception` decorator, that logs the exception, and it's error message and returns a `Http Response Object` 
+## HandleExceptions Decorator
+`Core-Lib`'s `web_helpers` provides `HandleExceptions` decorator, that logs the exception, and it's error message and returns a `Http Response Object` 

```

Comment:
add something about the status code. please use grammerly
`with error message and appropriate status code. `